doc/kernel/data_passing/message_queues.rst

```diff
@@ -55,6 +55,12 @@ Any number of receiving threads may wait simultaneously when the ring buffer
 is empty; when a data item becomes available it is given to
 the highest priority receiving thread that has waited the longest.
 
+A data item can also be **read** from a message queue without removing
```

Comment:
I'd simplify and replace this sentence (three lines) with:

    You can also peek at the message on the head of a message queue
    without removing it from the queue.